qiskit/transpiler/preset_passmanagers/generate_preset_pass_manager.py

```diff
@@ -226,7 +226,7 @@ def generate_preset_pass_manager(
                 raise ValueError(
                     f"Gates with 3 or more qubits ({gate}) in `basis_gates` or `backend` are "
                     "incompatible with a custom `coupling_map`. To include 3-qubit or larger "
-                    " gates in the transpilation basis, use a custom `target` instance instead."
```

Comment:
Isn't lowercase more correct to use here since that refers to the input argument? 🙂